tarantool_test.go

```diff
@@ -4068,6 +3872,107 @@ func TestFdDialer(t *testing.T) {
 	require.Equal(t, int8(0), resp[0])
 }
 
+func TestGetSchema_ok(t *testing.T) {
```

Comment:
Please, move unit-tests to schema_test.go . tarantool_test.go is for integration tests with tarantool.